src/polytopes/hexahedron.jl

```diff
@@ -24,9 +24,3 @@ A hexahedron with points `p1`, `p2`, ..., `p8`.
 struct Hexahedron{Dim,T,V<:AbstractVector{Point{Dim,T}}} <: Polyhedron{Dim,T}
   vertices::V
 end
-
-function facets(hex::Hexahedron)
-  connec = connect.([(1,2,3,4),(5,6,7,8),(1,5,6,2),
-                     (2,6,7,3),(3,4,8,7),(1,5,8,4)], Quadrangle)
-  (materialize(c, hex.vertices) for c in connec)
-end
```

Comment:
Perhaps the `facets` function is a function that is more related to the polytope than it is to the convention? I think we should keep it in the original polytope file and implement it with the GMSH convention here if needed. That way, people can compare the conventions for the pyramid in a single place as opposed to opening multiple convention files and searching for pyramid.

Reply:
Makes sense. Do you mean the `connectivity` functions though? `facets` is no longer defined for each polytope.

Reply:
Perhaps I confused things again... I mean whatever function that is responsible for listing the connectivities of the polytope.

Reply:
Should we decide to do this, we'll have to include the conventions.jl file inside geometry.jl because all those polytope implementations will use whatever conventions are defined there. On the other end I feel like it puts even more stuff inside geometry.jl, which is already large. What are your thoughts on this?

Reply:
I think I am getting the idea now. But I still think the name `connectivity` should be something else, maybe the plural `connectivities` would be enough.